gemfiles/rails_7_0.gemfile

```diff
@@ -20,7 +20,7 @@ gem "yard"
 gem "spring"
 gem "spring-watcher-listen", "~> 2.0.0"
 gem "rails-controller-testing", ">= 1.0.1"
-gem "rails", "7.0.8"
+gem "rails", "7.0.8.1"
```

Comment:
I think it's a good idea to update the Appraisal file too. That way, when we regenerate the gemfiles, they won't have old versions of Rails.

Reply:
That's true. I'll update it on the other PR's as well